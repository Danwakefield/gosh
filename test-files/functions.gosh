foofunc() {
	echo "SUCCESS 1"
}

A="SUCCESS 2"
barfunc() {
	local A
	A="FAIL 2"
}

foofunc
barfunc
echo $A
