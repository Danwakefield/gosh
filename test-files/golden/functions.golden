SUCCESS 1
SUCCESS 2
